GAME/DELTAPAK.C

```diff
@@ -1136,15 +1142,43 @@ int Load_and_Init_Cutseq(int num)
 	return 0;
 }
 
-struct ITEM_INFO *cutseq_restore_item(int num)
+struct ITEM_INFO *cutseq_restore_item(int num)//2D738(<), 2DA20(<)
 {
-	S_Warn("[*cutseq_restore_item] - Unimplemented!\\n");
+	if(level_items > 0)
+	{
+		for(int i = 0; i < numnailed; i++)
```

Comment:
Needs to be pure C or won't compile for PS1.

int i; should be declared at top of method.